libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser/main.m

```diff
@@ -0,0 +1,50 @@
+//
+//  main.m
+//  libPhoneNumber-GeocodingParser
+//
+//  Created by Rastaar Haghi on 7/1/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import "NBGeocoderMetadataParser.h"
+#import <Foundation/Foundation.h>
+
+int main(int argc, const char *argv[]) {
+  @autoreleasepool {
+    NBGeocoderMetadataParser *metadataParser =
+        [[NBGeocoderMetadataParser alloc] init];
+    NSArray *dirPaths = NSSearchPathForDirectoriesInDomains(
+        NSDocumentDirectory, NSUserDomainMask, YES);
+    NSString *documents = dirPaths[0];
+    NSString *geocodingFolder =
+        [documents stringByAppendingFormat:@"/geocoding/"];
+    NSArray *languages = [[NSFileManager defaultManager]
+        contentsOfDirectoryAtPath:geocodingFolder
+                            error:NULL];
+    NSArray *textFilesAvailable;
+    for (NSString *language in languages) {
+      textFilesAvailable = [[NSFileManager defaultManager]
+          contentsOfDirectoryAtPath:[NSString stringWithFormat:@"%@%@",
+                                                               geocodingFolder,
+                                                               language]
```

Comment:
Extract this to a variable, it'll make the whole thing reads much better

Reply:
Done.